sonar-project.properties

```diff
@@ -0,0 +1,9 @@
+sonar.projectKey=handsontable_handsontable
+sonar.organization=handsontable
+sonar.projectName=Handsontable
+sonar.sources=src,wrappers
```

Comment:
@budnix I'm not sure if I understand it correctly - does this include only `src` from `handsontable`, but entire projects (including the config files, etc) for the wrappers? Should it be that way?

Reply:
It means that looking from the root projects only all files within `src` and `wrappers` (except `**/__tests__/**,**/test/**,**/*.spec.js,**/*.spec.ts`) will be analyzed.

Reply:
So `./wrappers/react/rollup.config.js` would be analyzed, but `./webpack.config.js` wouldn't?

Reply:
As far as that files do not interfere with the general code quality score I don't mind to analyzed that file as well. I didn't want to create a long exclusion list.

Reply:
I guess after moving HOT to a subdirectory, we'll change it to `handsontable, wrappers`.

Reply:
Right 👌